loomi/regulariza-diagonal-client
# Feature/stamp list

## Feature
Stamps List
---

**What I did:**
-  Create Stamps List component
- Refactor component ResidentsTable to Table
- Add a font size type in theme

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/32523401/126837295-32099ff2-d289-4b2d-8920-1665807f98fc.png)
